src/execute_request.jl

```diff
@@ -133,7 +134,7 @@ function helpcode(code::AbstractString)
         @compat if !haskey(Docs.keywords, Symbol(code_))
             return "Base.Docs.@repl $code_"
         elseif VERSION < v"0.5.0-dev+3831"
-            return "eval(:(Base.Docs.@repl \$(symbol(\"$code_\"))))"
+            return "eval(:(Base.Docs.@repl \$(Symbol(\"$code_\"))))"
```

Comment:
It is written this way since there's already a branch and I was too lazy to figure out an order that `eval`, `@repl` and `@compat` are all happy about and that the `@compat` is evaluated in the right module.